docs/metadata/git.md

```diff
@@ -1,72 +1,67 @@
 ---
 title: Git
-id: git
-description: Source control
+id: Git
+description: Source Control
 sidebar_position: 1
 tags:
   - metadata
-  - git
+  - Git
 ---
 
 ## How to connect to external Git
 
-To connect to an external Git provider, Prophecy just need `Email`, `Username` and `Access Token`.
+Configure your user's [Settings](https://app.prophecy.io/metadata/settings) with a new `Git` provider: enter `Email`, `Username` and `Personal Access Token`, or simply `Login with Github`.
 
-![GitConnector](img/git_connector.png)
+![GitConnector](./img/git_connector.png)
 
-For Github Cloud we also have a first class integration to directly connect to your Github account by using `Login with Github` option.
-
-Please refer below video for step-by-step example for how to set up your external Git provider:
+The video below walks through connecting to an external Git provider:
 
 <div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
 <div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
-<iframe src="https://user-images.githubusercontent.com/121796483/216262458-a2c4a45b-d999-4fe8-99eb-f39a494062ca.mp4" title="External Git" allow="autoplay;fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+<iframe src="https://user-images.Githubusercontent.com/121796483/216262458-a2c4a45b-d999-4fe8-99eb-f39a494062ca.mp4" title="External Git" allow="autoplay;fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
 </div></div>
 
 Every user will have a `Prophecy Managed` Git provider already created in their account. You can just use this if you don't want to store your code to an external Git.
 
-## How to create a new project with external Git provider
-
-Once a Git Credential is created, these credentials can be used during project creation.
+## How to create a new Project with external Git provider
 
-Please refer below video for step-by-step example
+Once a Git Credential is created, these credentials can be used during Project creation (video below). Later, these credentials will be used to commit and release changes to Git for the entire Project.
```

Comment:
"Once a Git Credential is created, these credentials can be" -> "Once a Git Credential is created it can be..."